script/deploy-canary-demo

```diff
@@ -12,79 +13,131 @@ ChimeAlpha)
+    npm run deploy -- -r eu-central-1 -b chime-sdk-$chimeSDKMediaPipelinesStackId-demo-gamma-canary-eu-central-1$canarySuffix -s chime-sdk-meetings-demo-gamma-canary-eu-central-1$canarySuffix -u false -o chime-sdk-meetings-fra-demo-gamma-canary$canarySuffix -t -l --chime-sdk-media-pipelines-region eu-central-1 --chime-sdk-media-pipelines-endpoint $GAMMA_CHIME_MEDIA_PIPELINES_ENDPOINT_EU_CENTRAL_1
+    npm run deploy -- -r ap-southeast-1 -b chime-sdk-$chimeSDKMediaPipelinesStackId-demo-gamma-canary-ap-southeast-1$canarySuffix -s chime-sdk-meetings-demo-gamma-canary-ap-southeast-1$canarySuffix -u false -o chime-sdk-meetings-sin-demo-gamma-canary$canarySuffix -t -l --chime-sdk-media-pipelines-region ap-southeast-1 --chime-sdk-media-pipelines-endpoint $GAMMA_CHIME_MEDIA_PIPELINES_ENDPOINT_AP_SOUTHEAST_1
     ;;
 
 ChimeProd)
-    echo "Deploying to prod stage for canary that talks to prod Chime client"
+    echo "Deploying to prod stage for canary that talks to prod Chime client for meetings and prod Chime client for media pipelines"
 
     # Uses Chime Client
-    npm run deploy -- -b chime-sdk-demo-prod-canary$canarySuffix -o chime-sdk-demo-prod-canary$canarySuffix -s chime-sdk-demo-prod-canary$canarySuffix -u false -t -l
+    npm run deploy -- -b chime-sdk-demo-prod-canary$canarySuffix -o chime-sdk-demo-prod-canary$canarySuffix -s chime-sdk-demo-prod-canary$canarySuffix -u false -t -l --use-chime-sdk-media-pipelines false
     npm run deploy -- -b chime-sdk-meeting-readiness-checker-prod-canary$canarySuffix -s chime-sdk-meeting-readiness-checker-prod-canary$canarySuffix -a meetingReadinessChecker -u false -t -l
     ;;
 
 ChimeSDKMeetingsProdIAD)
-    echo "Deploying to prod stage for canary that talks to prod IAD ChimeSDKMeetings client"
+    echo "Deploying to prod stage for canary that talks to prod IAD ChimeSDKMeetings client for meetings and prod Chime client for media pipelines"
+
+    # Uses ChimeSDKMeetings client
+    npm run deploy -- -r us-east-1 -b chime-sdk-meetings-demo-prod-canary-us-east-1$canarySuffix -o chime-sdk-meetings-iad-demo-prod-canary$canarySuffix -s chime-sdk-meetings-demo-prod-canary-us-east-1$canarySuffix -t -l --use-chime-sdk-media-pipelines false
+    ;;
+
+ChimeSDKMeetings_ChimeSDKMediaPipelinesProdIAD)
+    echo "Deploying to prod stage for canary that talks to prod IAD ChimeSDKMeetings client for meetings and prod IAD ChimeSDKMediaPipelines client for media pipelines"
+
+    # Uses ChimeSDKMeetings client
+    npm run deploy -- -r us-east-1 -b chime-sdk-meetings-$chimeSDKMediaPipelinesStackId-demo-prod-canary-us-east-1$canarySuffix -o chime-sdk-meetings-iad-demo-prod-canary$canarySuffix -s chime-sdk-meetings-demo-prod-canary-us-east-1$canarySuffix -t -l --chime-sdk-media-pipelines-region us-east-1 --chime-sdk-media-pipelines-endpoint https://media-pipelines-chime.us-east-1.amazonaws.com
```

Comment:
Do we need to specify the `--chime-sdk-media-pipelines-endpoint` here, since `https://media-pipelines-chime.us-east-1.amazonaws.com` is the default in `deploy.js`?